test/control-plane/services/dual-stack/generate.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#
+# Generate the golden test files for the DualStack test
+#
+
+set -eux
+
+export KUBECONFIG=kubeconfig
+
+manifests=(
+	# Creates the echo deployment with two echo replicas.
+	manifests/echo-dpl.yaml
+
+	# Allows all to echo.
+	manifests/echo-policy.yaml
+
+	# Creates dual-stack service for echo.
+	manifests/echo_svc_dualstack.yaml
+)
+
+: Start a kind cluster
+kind create cluster --config kind-config.yaml --name dual-stack
```

Comment:
Where can we specify the k8s version?

Reply:
Good that you pointed this out. Especially for the service load-balancing we should have variants for the different k8s major versions due to the Endpoints/EndpointSlices stuff.

Kubernetes version can be set in kind config by using a specific image:
https://kind.sigs.k8s.io/docs/user/configuration/#kubernetes-version,
e.g. `image: kindest/node~v1.21.12` etc.

I'll take a look adding tests for different versions and hard-coding
the version in the generation scripts.